orgchart.md

```diff
@@ -8,11 +8,11 @@ Below you’ll find a list of the major groups and teams within the company. The
 
 **Andrea LaRowe** is Basecamp’s Head of People Ops. Andrea works with Jason and David to make working at Basecamp an excellent experience for its employees. She manages the day-to-day operations of the company, human resources, benefits & payroll administration, and event planning. Andrea is often the best person to tap first, when you don’t know who to talk to about a problem.
 
-Navid Afshar is Basecamp’s admin assistant. He assists Andrea with all administrative operations, and he’s primarily responsible for bookkeeping, supporting tax-related work, and coordinating the details for our meetups. Navid is another great person to ping if you don’t know where to go with a question.
+Navid Afshar was Basecamp’s admin assistant. He assisted Andrea with all administrative operations, and he was primarily responsible for bookkeeping, supporting tax-related work, and coordinating the details for our meetups. Navid was another great person to ping if you don’t know where to go with a question.
 
 ### Customer Support (aka Team OMG)
 
-**Kristin Aardsma** is the head of the customer support team. She’s responsible for the entire customer support team which uses [Help Scout](https://www.helpscout.net) to respond to customer tickets. Her team consists of Jabari Allen, Ashley Bowe, Haas Cavazos, Sylvia Chong, Chase Clemons, Merissa Dawson, Shanae Dykes, Tony Giang, James Glazebrook, Elizabeth Gramm, Chris Joyce, Lexi Kent-Monning, Marcia Deidra Boland, Jim Mackenzie, Willow Moline, Jayne Ogilvie, and Joan Stewart.
+**Kristin Aardsma** was the head of the customer support team. She was responsible for the entire customer support team which uses [Help Scout](https://www.helpscout.net) to respond to customer tickets. Her team consisted of Jabari Allen, Ashley Bowe, Haas Cavazos, Sylvia Chong, Chase Clemons, Merissa Dawson, Shanae Dykes, Tony Giang, ~~James Glazebrook~~, Elizabeth Gramm, Chris Joyce, ~~Lexi Kent-Monning~~, Marcia Deidra Boland, Jim Mackenzie, ~~Willow Moline~~, Jayne Ogilvie, and Joan Stewart.
```

Comment:
Shanae is no longer at basecamp according to Andy Baio https://twitter.com/waxpancake/status/1388268459256020993